recipe/meta.yaml

```diff
@@ -1,20 +1,57 @@
 {% set name = "bash" %}
-{% set version = "4.4.18" %}
+{% set _patch_level = 11 %}
+{% set base_version = "5.0" %}
```

Comment:
Can you make these two derived from version, so that the autotick bot can work with this?